container_files/guac-demo-compose.yaml

```diff
@@ -53,3 +53,27 @@ services:
         condition: service_healthy
       graphql:
         condition: service_healthy
+
+  guac-rest:
+    image: $GUAC_IMAGE
+    command: "/opt/guac/guacrest --rest-api-server-port=8081 --gql-addr=http://graphql:8080/query"
+    working_dir: /guac
+    restart: on-failure
+    ports:
+      - "8081:8081"
+    depends_on:
+      graphql:
+        condition: service_healthy
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "wget",
+          "--spider",
+          "http://localhost:8222/healthz"
```

Comment:
is this healthcheck valid? port 8222?

Reply:
oops I thought I fixed that in both places. Let me fix it now